readme/change_codes.md

```diff
@@ -1,192 +1,360 @@
 # Reasons for detected changes
 
-dart-apitool has a set of rules it applies in order to detect (breaking) changes. 
+dart-apitool has a set of rules it applies in order to detect (breaking) changes.
 Those are often not clear at the first glance this is why this page tries to explain the different situations that lead to (breaking) changes and dart-apitool complaining about the selected version.
 
-The general rule of thumb: If a change can break a user's code without the user changing something (but using semver) then we have a breaking change. 
+The general rule of thumb: If a change can break a user's code without the user changing something (but using semver) then we have a breaking change.
 Therefore, we only consider changes in the public API of a package (meaning: things reachable without including stuff from the src directory).
 
 ## Interfaces
-Dart-apitool treats classes, interfaces (abstract classes), enums, mixins and extension classes as interfaces.
 
-### <a name="CI01" />An interface is removed (CI01)
+Dart-apitool treats classes, interfaces (abstract classes), enums, mixins, and extension classes as interfaces.
+
+### CI01
+
+> An interface is removed
+
 This is a breaking change because it is possible that a user of the library is using the interface. If the interface is removed, the user will get a compile error.
 
-### <a name="CI02" />An interface is added (CI02)
+### CI02
```

Comment:
is there a special reason for removal of the anchors?

Reply:
When trying using the anchors in the URL it didn't work for me atleast, so I decided to nest them differently so I can link from the Markdown report to the correct section, however if you know another solution to make the "old" one work we can surely use that!

Reply:
Yes, makes sense. 👍